docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: '3.2'
+services:
+  kbs:
+    # build:
+    #   context: .
+    #   dockerfile: ./docker/Dockerfile.grpc-as
+    image: quay.io/confidential-containers/key-broker-service:v0.5.0-rc3
```

Comment:
The container images of AS, RVPS and KeyProvider are all with `-rcX` suffix. We can build the final version of these images after code freeze and update here into `<componant-name>:v0.5.0` tag

cc @sameo @jialez0